backend/flaskr/users/handlers.py

```diff
@@ -20,22 +22,30 @@ def make_token():
     if error_res is not None:
         return error_res
 
-    # TODO: Use 'username' to get user from db
-    #       then perform password check.
     # Token should be issued only after successful check.
+    # Check if user exists
+    username = body["username"]
+    user = UserD.get_user({"username": username})
+    if user is None:
+        return "There is no user with this name!"
```

Comment:
This gives 'text/plain', non-JSON string response.

It should be:

mk_error("There is no user with this name!")()